### DISCUSSION TITLE
adding parts per million as a unit option to the air quality concentration

### DISCUSSION THREAD

**Discussion by GreencapGonzo** [breezy-weather/breezy-weather#1230] [Ideas]
### Describe your idea

While I understand if it is not in your current scheme for the app, it would be really nice if you had an option to display the air quality index in PPM. As someone who is far more familiar with imperial units rather than metric, I'm afraid that nanograms per cubic meter does not mean very much to me.

### Other details

The air quality index is an important thing for me to know as someone who has minor respiratory problems and who enjoys being outside when he is able. 

### Acknowledgements

- [X] I have searched the [existing issues](https://github.com/breezy-weather/breezy-weather/issues) and [existing discussions](https://github.com/breezy-weather/breezy-weather/discussions/categories/ideas) and this is a new ticket, **NOT** a duplicate or related to another open OR closed issue or discussion.
- [X] The idea I’m suggesting is not in the [list of features that will not be implemented](https://github.com/breezy-weather/breezy-weather/blob/main/README.md#features-that-will-not-be-implemented).
- [X] I have updated the app to **[version 5.2.6 or later](https://github.com/breezy-weather/breezy-weather/releases/latest)**. Even if I’m getting updates from F-Droid default repo, I don’t need to uninstall/reinstall to update.
- [X] I wrote my request in English, and any screenshot I attached are also in English (language can be changed from Settings > Appearance). I can write in my native language under the English text, if necessary.
- [X] I understand that this app is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY, so I may not get a positive reply to my request, if any at all.

**Comment by papjul** [comment]
> nanograms per cubic meter does not mean very much to me

> The air quality index is an important thing for me

Air quality concentration and air quality index are two different things. If you are looking for meaningful information about air quality, we have a very easy to understand air quality index. Have a look at the documentation: https://github.com/breezy-weather/breezy-weather/blob/main/docs/HOMEPAGE.md#current-air-quality

This is probably easier to understand than PPM or ug/m³. Especially as converting between the two is more complex than other units. It is a different formula for each molecule, and depends on the temperature, so we can't do the same thing we do with other units like temperatures or distances.
